package-lock.json

```diff
@@ -1592,6 +1592,14 @@
         "fromentries": "^1.2.0"
       }
     },
+    "randombytes": {
+      "version": "2.1.0",
+      "resolved": "http://nexus3.ealan.shinesolutions.com:8081/repository/npm-all/randombytes/-/randombytes-2.1.0.tgz",
```

Comment:
re-generating lock file with `registry.npmjs.org` would be nice.

Reply:
sorry about that - should be fixed now